packages/beacon-node/src/chain/archiveStore/utils/xdelta3.ts

```diff
@@ -0,0 +1,20 @@
+import {decodeSync, encodeSync} from "@chainsafe/xdelta3-node";
+import {IStateDiffCodec} from "../interface.js";
+
+export class XDelta3Codec implements IStateDiffCodec {
```

Comment:
```suggestion
export class BinaryDiffCodec implements IStateDiffCodec {
```

don’t think developers need to know about xdelta3 in our code base. If in the future we are adopting another binary diff codec we can revisit the class naming